internal/command/postgres/config_show.go

```diff
@@ -103,7 +103,12 @@ func runMachineConfigShow(ctx context.Context, app *api.AppCompact) (err error)
 		return err
 	}
 
-	return showSettings(ctx, app, leader.PrivateIP)
+	manager := flypg.StolonManager
+	if leader.ImageRef.Repository == "flyio/postgres-flex" {
```

Comment:
still wish there was a flag or something so when I am developing and my image comes from my personal repo it could still work. But also I'm fine just commenting stuff out for now

Reply:
Yeah, we can do that.  What about something like `PG_MANAGER`?

Reply:
Yeah I can open a different PR for it if you want